internal/pkg/clik8s/clik8s.go

```diff
@@ -32,4 +32,7 @@ type KubeConfigPath string
 type MasterURL string
 
 // ResourceConfigs is a collection of Resource Config read from Files.
-type ResourceConfigs []runtime.Object
+type ResourceConfigs []unstructured.Unstructured
+
+// ResourcePruneConfigs is a collection of Resource Config used for pruning
+type ResourcePruneConfigs []unstructured.Unstructured
```

Comment:
Lets make this a single pointer for now - e.g. *unstructured.Unstructured

Reply:
updated